oauth2client/contrib/gce.py

```diff
@@ -51,13 +59,17 @@ class AppAssertionCredentials(AssertionCredentials):
     """
 
     @util.positional(2)
```

Comment:
Continuing the discussion started in #411: I'm not entirely comfortable with this decorator, but I think the right thing for now is to leave this as it is.